packages/application-generic/src/services/queue.service.ts

```diff
@@ -31,14 +34,16 @@ export class QueueService<T = unknown> {
     });
   }
 
-  public async gracefulShutdown() {
-    // Right now we only want this for testing purposes
-    if (process.env.NODE_ENV === 'test') {
-      await this.bullMqService.queue.drain();
```

Comment:
With the new change we will be missing the 'queue.drain()', is that ok on the testing env?

Reply:
I was the one implementing that as a hack to make one Workflow Queue test pass. Seems that hack is not needed anymore.